app/controllers/home_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
     @rentals = if @current_user.groups.where(name: 'admin').present?
                  Rental.all
                else
-                 @current_user.rentals
+                 Rental.where(renter: @current_user)
```

Comment:
not any more it will not, `user.rentals` will retrieve a list of all associated rentals for that user including ones they reserved and the ones that they are the renter.  There are two fields in `Rental` that refer to users, `renter_id` and `creator_id`.  The is often different than the renter, for example when a csr creates it.